docs/kyma/docs/040-cr-installation.md

```diff
@@ -37,3 +49,7 @@ This table lists all the possible parameters of a given resource together with t
 | **metadata.finalizers** | **NO** | Protects the CR from deletion. Read [this](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) Kubernetes document to learn more about finalizers. |
```

Comment:
As discussed, pls add the third option `update` in the **metadata.labels.action** field description.

Reply:
done